test/acceptance/display-conditions.spec.js

```diff
@@ -70,6 +70,40 @@ describe('when setting display conditions', function () {
     }, 100);
   });
 
+  it('should correctly calculate scroll percentage when scroll offset cannot be greater than scroll position', function (done) {
+    $(document.body).append(
+      '<div id=\'height-element\' style=\'height:800px; display:block;\'>Test</div>'
+    );
+
+    var subscription = new pathfora.Message({
+      layout: 'modal',
+      id: 'scrollModal',
+      headline: 'Heyyyy!',
+      msg: 'You have scrolled far, nice work.',
+      okMessage: 'Sure, whatever',
+      okShow: true,
+      displayConditions: {
+        scrollPercentageToDisplay: 30
+      }
+    });
+    pathfora.initializeWidgets([subscription]);
+
+    var widget = $('#' + subscription.id);
+    expect(widget.length).toBe(0);
+
+    var height = $(document.body).height();
+    window.scroll(0, height);
```

Comment:
Maybe we should try a partial scroll on this test too? Since the goal was to fix the math here? :thinking:

Reply:
Good idea. This test now scrolls `height / 2`.